examples/textConnect/contents/setting.html

```diff
@@ -4,210 +4,218 @@
+          <span class="kintoneplugin-require">*</span>
+      </label><br >
+      <div class="kintoneplugin-desc">{{html:resultDescription}}</div>
+      <div class="kintoneplugin-row">
+          <span class="num">①</span>
+          <div class="kintoneplugin-select-outer">
+            <div class="kintoneplugin-select">
+              <select name="copyfield" id= "copyfield1">
+                <option value="">-----</option>
+              </select>
+            </div>
+          </div>
+      </div>
+      <div class="kintoneplugin-row">
+          <span class="num">②</span>
+          <div class="kintoneplugin-select-outer">
+            <div class="kintoneplugin-select">
+              <select name="copyfield" id= "copyfield2">
+                <option value="">-----</option>
+              </select>
+            </div>
+          </div>
+      </div>
+      <div class="kintoneplugin-row">
+      <span class="num">③</span>
+          <div class="kintoneplugin-select-outer">
+            <div class="kintoneplugin-select">
+              <select name="copyfield" id= "copyfield3">
+                <option value="">-----</option>
+              </select>
+            </div>
+          </div>
+      </div>
     </div>
-    <div class="kintoneplugin-row" id="row2">
-        <span class="num">②</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select6" id= "select6">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select7" id= "select7">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select8" id= "select8">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select9" id= "select9">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
-        <span class="plus">＋</span>
-        <div class="kintoneplugin-select-outer">
-          <div class="kintoneplugin-select">
-            <select name="select10" id= "select10">
-              <option value="">-----</option>
-            </select>
-          </div>
-        </div>
+    <br >
+    <div class="block">
+      <button type="button" id="submit" class="kintoneplugin-button-dialog-ok">     Save     </button>
+      <button type="button" id="cancel" class="kintoneplugin-button-dialog-cancel">     Cancel   </button>
```

Comment:
Please localize the Save Cancel buttons as well (this currently only displays it in English, no matter what language you are looking at it in)

Reply:
Updated the codes. Please see the latest commit made on 8/17.